test/extensions/Worker.Extensions.Http.AspNetCore.Tests/HttpResultAttributeExpectedTests.cs

```diff
@@ -0,0 +1,219 @@
+﻿using AnalyzerTest = Microsoft.CodeAnalysis.CSharp.Testing.CSharpAnalyzerTest<Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore.HttpResultAttributeExpectedAnalyzer, Microsoft.CodeAnalysis.Testing.Verifiers.XUnitVerifier>;
```

Comment:
Just noticed that none of the analyzer tests have the msft license but they all should, can you add it to this file for now and a new PR can be created to address the rest.